blog/docs/events/virtual-office-hour-12-2022.md

```diff
@@ -0,0 +1,52 @@
+---
+title: "First 🌱 Knative Virtual Office Hour / Contrib Fest 2022/12/09"
+type: "blog"
+date: 2022-12-01
+linkTitle: "First Knative Virtual Office Hour / Contrib Fest 2022/12/09"
+description: "Join the Zoom Knative First Knative Virtual Office Hour / Contrib Fest on 2022/12/09"
+---
+
+## First 🌱 Knative Virtual Office Hour / Contrib Fest
+
+We think that there should be an easy way for new Knative users and contributors to get started with our project. Holding virtual office hours for questions and help with using or contributing to Knative should help to get started with using code and contributing.
+
+During the recent KubeCon NA 2022 in Detroit we had Knative Day and ContribFest that showed that there are people interested in using Knative or contributing to it. However, today we do not have any ongoing online meetings for people to just show up. They can join Slack but that is very asynchronous and joining yet another Slack and then reading channels, figuring where to ask your questions, etc. feels very daunting,
+
+Therefore, we are organizing the first Knative Virtual Office Hour / Contrib Fest in 2022.
+
+**When:** [December 9 (Friday) 11:30 AM US Eastern](https://dateful.com/convert/utc?t=430pm&d=2022-12-09)
+**How to join:** Zoom link will be posted in Slack and knative-user mailing list
+
+**Who should join?**
+
+- You are curious about Knative and you are wondering where to start
+- You would like to contribute to Knative but do not know where to start
+- You have problems with Knative and asking in Slack or Github issues did not get you what you wanted
```

Comment:
I'd remove the "asking in Slack or GitHub", that feels like it might shame people into not attending. We should steer them _after_ they show up, rather than making them feel like they can't show up. 😁